### PR TITLE
Add translation process, guidelines, working with rst files to Style Guide

### PR DESCRIPTION
**Pull request by catong**
Create files for doc translation guidelines, process, and working with rst files, and add to style guide.
Includes a sample rst file that is not included in Style Guide output - but should be accessible via a link from the style guide. (https://openedx.atlassian.net/browse/DOC-1837)
@mhoeber @srpearce @lamagnifica please take a look. This will be a WIP for a while, but at least we should be able to get something usable out.
Draft HTML is here: http://draft-translation-guide-wip.readthedocs.org/en/latest/DocTranslationProcess.html

### CODE REVIEW

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. _Documentation Translation Guidelines:
+
+###############################################
+Guidelines for Translators
+###############################################
+
+This section provides basic guidelines for translators of edx documentation.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: edX

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+The complete edX documentation translation process is described in
+:ref:`Documentation Translation Process`.
+
+.. note:: Before you begin an edX translation project, you should have a basic
+   understanding of restructuredText and formatting in .rst files. For more
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: capital S in reStructuredText

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+   information, see :ref:`Work with edX Documentation Source Files`.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: possibly add external references to rst as well as to GitHub? http://docutils.sourceforge.net/rst.html would be my suggestion

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+In addition to obvious text, do translate the following items.
+
+* Note and Warning text. Translate text following the tags ``.. note::`` or
+  ``.. warning::`` Be careful to wrap and indent lines correctly.
+
+* Link text in inline links. Translate text following the tag ``:ref:``
+  Be careful not to modify the referenced link anchor.
+* Alt text in images. Translate text following the tag ``:alt:``.
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst:71]: examples of each of these will be helpful -- in the future

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+*****************************
+
+Do not translate or alter any of these elements.
+
+* The filenames and locations of any .rst files in the repository.
+* Any part of the .rst tags listed in :ref:`Work with edX Documentation Source
+  Files` and :ref:`SampleRSTFile<Anchor For SampleRSTFile>`. 
+* File paths in image references. 
+* The text of anchors in .rst files. 
+* Code examples that are tagged with ``..code-block::``.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: missing space after ..

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+* The filenames and locations of any .rst files in the repository.
+* Any part of the .rst tags listed in :ref:`Work with edX Documentation Source
+  Files` and :ref:`SampleRSTFile<Anchor For SampleRSTFile>`. 
+* File paths in image references. 
+* The text of anchors in .rst files. 
+* Code examples that are tagged with ``..code-block::``.  
+* Variables or database field names that are tagged with
+  monospace code font (double grave accent characters).
+
+For information about working with edX Documentation source files, see
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst:105]: lc Documentation

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+What To Translate
+*************************
+
+In addition to obvious text, do translate the following items.
+
+* Note and Warning text. Translate text following the tags ``.. note::`` or
+  ``.. warning::`` Be careful to wrap and indent lines correctly.
+
+* Link text in inline links. Translate text following the tag ``:ref:``
+  Be careful not to modify the referenced link anchor.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: this one is a bit tricky -- we do want them to translate text IF the link is provided after it within `< >` characters, otherwise not, correct? 
so, 
yes, translate "course start date" in a link that looks like this
:ref:`course start date<The Course Start Date>`
but no, do not translate a link that looks like
:ref:`The Course Start Date` OR  `The Course Start Date`_

an interesting follow on point is, do we want them to translate the bookmarks? I think not, since translating the header that follows should "just work".

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 8]
```diff
@@ -0,0 +1,63 @@
+.. _Documentation Translation Process:
+
+#########################################
+Documentation Translation Process
+#########################################
+
+Before you begin an edX documentation translation project, you should have a
+basic understanding of restructuredText and formatting in .rst files. For more
+information, see :ref:`Work with edX Documentation Source Files`.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst:14]: does this (these 2 paragraphs) need to be repeated? if so, make the wording exactly the same in both places

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+In addition, you or another person on the translation team should understand
+Github repositories and the Github pull request process. For more information
+about using Github, see https://help.git.com. You or another person on the
+translation team should understand Github repositories and the Github pull
+request process. For more information, see https://help.git.com.
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst:19]: repeated info ^^

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+To translate edX documentation, follow these steps.
+
+#. Use Github to clone the ``edx-documentation`` repository to a location on
+   your local file system.
+
+#. Language folders, including ``en_us``, are placed one level below
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: I suggest making a separate section on what the directory structure looks like. Then this step could have the verb closer to the beginning of the sentence, like:
#. If this is your initial translation project, duplicate the entire `en_us` directory in the repository at the same level, and then rename the directory to reflect your language and locale. For more information about the expected directory structure, see new topic name.

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+   ``edx-documentation``. If the language you are translating into does not already
+   exist, duplicate the entire ``en_us`` structure in the repository to the
+   same level. Then, rename only the ``en_us`` folder with your translation
+   target language, using ISO standard language-locale pairs, such as
+   ``fr_ca``,``fr_fr``, ``zh_tw``, ``zh_cn``, and so on. The resulting folder
+   is your working source directory.
+
+#. Follow best practices for Github use. For information, see https://help.github.com.
+
+	* Create a working branch for your changes. Before making new branches,
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: I think they would create the branch first, and then duplicate / create the whole new language directory(?) Possibly, pull this step out of the procedure and make it part of the intro section above?

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+	* On your branch(es), make edits to the source files using your preferred
+	  text editor. 
+	* Save changes in the source files without renaming or moving the files.
+	* Commit changes back to Github. 
+	* Create pull requests in Github for review purposes.
+
+4. Build draft output and make sure everything renders as it should in both
+   HTML and PDF formats. 
+
+#. Notify the edX Documentation team when you have changes ready for review
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst:57]: how?

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+   and testing. The team might also be able to help you to generate draft
+   output for review and testing.
+
+#. Make any revisions resulting from reviews and testing, then notify the edX
+   Documentation team that the pull request is ready to merge into the repository.
+
+#. The edX Documentation team merges translation pull requests into the
+   repository.
+
+#. The edX Documentation team builds updates documentations projects for the
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: reword?

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+   next release or a planned future release.
+
+
+For information about working with edX Documentation source files, see
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst:71]: lc Documentation? not sure what your strategy is here -- uc the team name?

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 15]
```diff
@@ -0,0 +1,214 @@
+.. _Anchor For SampleRSTFile:
+
+#################
+Sample .rst File
+#################
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: describe the content/purpose/audience of this file here (future)

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 16]
```diff
@@ -0,0 +1,214 @@
...
+
+.. _Anchor for Heading One:
+
+################################
+Heading One has hash characters
+################################
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: I read most of this before I realized that this is an example of a first level heading, which is preceded and followed by # characters!
Maybe have more descriptive H1 text?

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 17]
```diff
@@ -0,0 +1,214 @@
...
+
+Headings are text sandwiched between rows of specific characters. The row of
+hash, asterisk, or equals characters must be as long as, or longer than, the
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: show what you mean for each of these (a lot of people say pound instead of hash, for example)

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 18]
```diff
@@ -0,0 +1,214 @@
...
+#################
+
+
+.. _Anchor for Heading One:
+
+################################
+Heading One has hash characters
+################################
+
+Headings are text sandwiched between rows of specific characters. The row of
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: row -->  rows

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 19]
```diff
@@ -0,0 +1,214 @@
...
+hash, asterisk, or equals characters must be as long as, or longer than, the
+text of the heading. If the translated heading string is longer than the
+original English source, make sure you extend both lines above and below the
+heading string.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: they also have to be the same length as each other...

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 20]
```diff
@@ -0,0 +1,214 @@
...
+
+This is regular text in paragraph form. There are no indents. As a best
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: maybe add a transition sentence or clause? It was hard to parse this information, since without its own heading I thought we were still talking about heading 1s...

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 21]
```diff
@@ -0,0 +1,214 @@
...
+practice, break lines at about 80 characters, so that each line has its own
+line number for commenting in reviews. This is regular text in paragraph form.
+There are no indents. As a best practice, break lines at about 80 characters,
+so that each line has its own line number for commenting in reviews.
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: repeated info ^^

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 22]
```diff
@@ -0,0 +1,214 @@
...
+.. comments can be added in a file by starting a line with 2 periods and a space. 
+
+In English source files, look for comments addressed to translators from writers.
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: .. Translators: For example, ...
say something about whether or not they should translated the comments themselves?

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 23]
```diff
@@ -0,0 +1,214 @@
...
+.. warning:: Warnings are formatted in the same way as notes. In the same way,
+   lines must be broken and indented under the warning tag.
+
+
+.. _Anchor for Heading Two:
+
+************************************
+Heading Two has asterisk characters
+************************************
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: maybe it's just me, but I'd put all the example headings together at the beginning with a description. Then, actually use the different heading to introduce discussions of the different components: notes and warnings, lists, applying formatting, links, ...
Ah! maybe structure in the same way as the WorkingWithEdXDocSource.rst file -- make them parallel.
I'm not entirely sure that both files are needed...

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 24]
```diff
@@ -0,0 +1,214 @@
...
+
+#. Switch to the **HTML** tab. 
+
+#. Replace the following placeholders with your own content.
+
+   * Replace the value of the <a> element's href attribute with the path to
+     your image. Do not change the value of the class attribute. For example:
+
+     **<a href="/static/Image1.jpg" class="modal-content">**
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: I think that should be mono rather than bold, but you probably just pasted this in from somewhere else...

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 25]
```diff
@@ -0,0 +1,214 @@
...
+Tables
+************************************
+
+Tables look like this. Each row is indicated with an asterisk. Each column in
+a row is indicated with a hyphen. The following table has three columns. There
+is a single space between the asterisk and the hyphen. Note that the asterisks
+must all align, and the hyphens must all align. Empty cells must be accounted
+for, so that each column in a row is always marked, even if there is no
+content in a table cell.
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: I don't think this is a great example, since the 2nd column seems to always be blank. Maybe some subset from the change log?

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 26]
```diff
@@ -0,0 +1,214 @@
...
+************************************
+Code Examples
+************************************
+
+===========
+Inline code
+===========
+
+In inline text, any text can be formatting as code (monospace font) by
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: formatting --> formatted

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 27]
```diff
@@ -0,0 +1,346 @@
...
+* :ref:`Documentation Translation Process`
+
+For an example .rst file containing examples of most of the formatting used in
+edX documentation files, see :ref:`SampleRSTFile.rst<Anchor For SampleRSTFile>`.
+
+
+.. _Documentation Tools and Processes:
+
+*********************************
+Documentation Tools And Processes
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst]: lc And

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 28]
```diff
@@ -0,0 +1,346 @@
...
+*********************************
+
+For information about the tools that the edX Documentation team uses to create
+documentation, see this wiki page:
+https://openedx.atlassian.net/wiki/display/DOC/Documentation+Tools
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst:30]: is this public? we need to revisit it to see what is actually useful for translators (or move the info into another part of this document)

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 29]
```diff
@@ -0,0 +1,346 @@
...
+.. _Comments:
+
+========
+Comments
+========
+
+Lines beginning with 2 periods and a space indicate comments that are not
+visible in output. For example:
+
+``.. This is a comment.``
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst]: I think that the `` characters need to be removed, or explained.

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 30]
```diff
@@ -0,0 +1,346 @@
...
+
+
+.. _Special Tags:
+
+============
+Special Tags
+============
+
+Special tagging for notes, warnings, tables, and images is achieved using
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst]: I'd change images --> blocks of code since you have a whole separate section on images, and one of your examples is a code-block

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 31]
```diff
@@ -0,0 +1,346 @@
...
+``.. code-block::``
+
+
+.. _Image Links:
+
+============
+Image Links
+============
+
+Images are included in documentation using special tagging for images and
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst]: "for images and providing" --> to provide

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 32]
```diff
@@ -0,0 +1,346 @@
...
+======
+Tables
+======
+
+Tables are tagged using ``.. list-table::``
+
+Each table has the number of columns and their associated relative widths
+indicated in a width tag.
+
+For proper formatting, the asterisk indicating each row must align, and the
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst]: align vertically?

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 33]
```diff
@@ -0,0 +1,346 @@
...
+
+For example, ``.. _Anchor Text:``
+
+.. note:: Anchor text is never visible in output. It is replaced either by the
+   actual text of the target heading, or by link text that you explicitly
+   specify. Do not edit or translate anchor text.
+
+------------------------------------------------------------
+Cross references using text of the target topic heading
+------------------------------------------------------------
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst]: per our Format Cheat Sheet.rstnouse file, our 3rd level headings just have a row of asterisks underneath them. 
is it possible to leverage that file more for this effort?

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 34]
```diff
@@ -0,0 +1,346 @@
...
+========
+Headings
+========
+
+The level of a heading is indicated by a series of characters above and below
+the heading text.
+
+* H1: pound symbol (#) 
+* H2: asterisk (*)
+* H3: equals symbol (=)
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst:90]: it's a row f asterisks underneath, actually

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 35]
```diff
@@ -0,0 +1,346 @@
...
+For example,
+::
+
+  For more information, see :ref:`the introductory section on
+  exercises<Exercises_introduction>```
+
+where ``Exercises_introduction`` is the text for an anchor that exists
+somewhere in the files that make up the guide, and “the introduction section
+on exercises” is the link text.
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst:315]: watch out for curly quotes. Also, mention that curly quotes and apostrophes are super evil in this document?

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 36]
```diff
@@ -0,0 +1,346 @@
...
+==============
+Code Examples
+==============
+
+---------------
+Inline Code
+---------------
+
+In inline text, any text can be formatting as code (monospace font) by
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst:327]: formatting --> formatted

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 37]
```diff
@@ -0,0 +1,346 @@
...
+=================
+
+Boldface is used for labels that are visible in the user interface. The UI
+text is surrounded by double asterisks. For example, **bold**.
+
+Italics are rarely used. Text surrounded by single asterisks is rendered in
+*italics*.
+
+Monospace text is used for ``code examples``. Text surrounded by double back
+quotation marks (`) is rendered in monospace font. Within the double back
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst]: suggest being consistent about what you call these

**Comment by lamagnifica** [comment]
This is a terrific start. I've corrected a couple of typos and made some suggesions for the future. I'm not sure how you envision the Format Cheat Sheet.rstnouse, SampleRSTFile.rst, and the other three files working together, though. I think they are pretty redundant (and occasionally contradictory) right now. That might be the only area that maybe needs to tighten up a bit before this is shared externally.

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst (hunk 3 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: Links to other rst resources are in the "Working with edX Documentation Source Files" file. This file is meant to be primarily about the translation process.

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst (hunk 7 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: Oops, thanks for the catch, I'll add the example. We definitely don't want anchors (bookmarks ) translated, and I think I make that clear elsewhere.

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst (hunk 4 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst:71]: I have to re-evaluate/consolidate the examples and make sure I provide links.

File: en_us/edx_style_guide/source/SampleRSTFile.rst (hunk 21 above)
Comment (catong) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: That was deliberate as paragraph text filler, but I don't think it's necessary to make the paragraph a certain length so I've now removed it.

File: en_us/edx_style_guide/source/SampleRSTFile.rst (hunk 22 above)
Comment (catong) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: I doubt that they should/would translate comments.

File: en_us/edx_style_guide/source/SampleRSTFile.rst (hunk 24 above)
Comment (catong) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: Yep, for some reason it was bold. I changed it to mono in this example.

File: en_us/edx_style_guide/source/SampleRSTFile.rst (hunk 25 above)
Comment (catong) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: I think a single example is adequate for getting the point across. A more complex example is probably nice to have but not necessary.

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 38]
```diff
@@ -0,0 +1,59 @@
...
+
+   In addition, you or another person on the translation team should
+   understand Github repositories and the Github pull request process. For
+   more information about using Github, see https://help.git.com.
+
+When you translate edX documentation source .rst files, follow these
+general guidelines.
+
+.. warning:: Do not rename or move folders or files (including .rst, .png).
+   The location and names of files in your language folder structure should
```
Comment (mhoeber) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: do we described the language folder structure?

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst (hunk 5 above)
Comment (mhoeber) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst]: also don't translate inline code `between these characters`

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 39]
```diff
@@ -0,0 +1,63 @@
...
+
+	* Create a working branch for your changes. Before making new branches,
+	  check that your version of “edx-documentation/master” is up to date. 
+	* On your branch(es), make edits to the source files using your preferred
+	  text editor. 
+	* Save changes in the source files without renaming or moving the files.
+	* Commit changes back to Github. 
+	* Create pull requests in Github for review purposes.
+
+4. Build draft output and make sure everything renders as it should in both
```
Comment (mhoeber) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: do we explain make html and make latexpdf anywhere?

File: en_us/edx_style_guide/source/SampleRSTFile.rst [hunk 40]
```diff
@@ -0,0 +1,220 @@
...
+
+.. image:: ../../../shared/building_and_running_chapters/Images/about_page.png
+   :alt: An image of the course summary page.
+
+Some image links might have additional specifications such as height, width,
+or scale.
+
+Alternative text for screen readers is required for each image. Provide text
+that is useful to someone who might not be able to see the image.
+
```
Comment (mhoeber) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: do we state somewhere that if you are providing a localized image, you must keep same file name and replace the english image?

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst (hunk 28 above)
Comment (mhoeber) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst:30]: I think tools pages should be public for translators and open source community

File: en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst [hunk 41]
```diff
@@ -0,0 +1,350 @@
...
+
+Create bulleted lists using the asterisk symbol followed by a period, for each
+item in the bulleted list. For example,
+
+* Who is teaching the course?
+* What university or college is the course affiliated with?
+* What topics and concepts are covered in your course?
+* Why should a learner enroll in your course?
+
+For both numbered and bulleted lists, ensure that wrapped lines are indented
```
Comment (mhoeber) [inline en_us/edx_style_guide/source/WorkingWithEdXDocSource.rst:139]: I'd add a note saying nested lists are supported, but you must check output because sometimes text blocks can change counts in numbered lists.

**Comment by mhoeber** [comment]
@catong  this is great, thank you.  a few questions in line

File: en_us/edx_style_guide/source/DocTranslationProcess.rst (hunk 39 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: I will add, or add link to a reference. Maybe future, as it will probably also involve documenting all the required config/setup files?

File: en_us/edx_style_guide/source/SampleRSTFile.rst (hunk 40 above)
Comment (catong) [inline en_us/edx_style_guide/source/SampleRSTFile.rst]: Good point! I will make sure to add this.

**Comment by catong** [comment]
Addressed review comments. Incorporated the old FormatCheatSheet into ExampleRSTFile.rst. With the most recent edits I hope it's clearer why there is one file that provides context, explanations, gotchas, etc. about working with our source files and a separate file that is intended to just provide examples of rst formatting. 
@mhoeber @lamagnifica I need to deliver the Style Guide with these updated files. Please take a look for anything egregious; (HTML version is here: http://draft-translation-guide-wip.readthedocs.org/en/latest/) 
I'm sure there's plenty to improve on in future, but for now I hope this makes sense and will provide value for the translation team.

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 42]
```diff
@@ -0,0 +1,104 @@
...
+
+* Code examples that are tagged with ``.. code-block::``.  
+
+* Variables, database field names, or code examples that are tagged with
+  monospace code font (placed between double grave accent characters). 
+
+  For example, in the following sentence, do not translate the event names that are between pairs of double grave accent characters.  ::
+
+  The edX mobile app for iOS now emits ````play_video````, ````pause_video````,
+  ````stop_video````, ````load_video````, and ````seek_video```` events.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst:102]: these event names should only have one pair around each one, not two pairs: `pause_video`

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 43]
```diff
@@ -0,0 +1,65 @@
...
+
+#. Language folders, including ``en_us``, exist one level below ``edx-
+   documentation``. If the language you are translating into does not already
+   exist, duplicate the entire ``en_us`` structure in the repository to the
+   same level. Then, rename only the ``en_us`` folder with your translation
+   target language, using ISO standard language-locale pairs, such as
+   ``fr_ca``,``fr_fr``, ``zh_tw``, ``zh_cn``, and so on. The resulting folder
+   is your working source directory.
+
+#. Follow best practices for Github use. For information, see https://help.github.com.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: link the URL?

File: en_us/edx_style_guide/source/DocTranslationProcess.rst (hunk 11 above)
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: I still think that they need to create a branch, then do their directory reorganization/duplication/renaming, and then make a PR for that change. Otherwise, they're making changes directly to master, aren't they?

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 44]
```diff
@@ -0,0 +1,65 @@
...
+
+   https://openedx.atlassian.net/wiki/display/DOC/Instructions+for+Updating+Documentation
+
+
+#. Notify the edX Documentation team when you have changes ready for review
+   and testing. The team can also help you generate draft output for review
+   and testing.
+
+#. Make any revisions resulting from reviews and testing, then notify the edX
+   Documentation team that the pull request is ready to merge into the repository.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst:62]: into --> in to (?)

File: en_us/edx_style_guide/source/ExampleRSTFile.rst [hunk 45]
```diff
@@ -0,0 +1,471 @@
...
+Code Formatting
+*****************
+
+===========
+Inline code
+===========
+
+In inline text, any text can be formatted as code (monospace font) by
+enclosing the selection within a pair of double "grave accent" characters (`).
+For example, ````these words```` are formatted in a monospace font when the
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/ExampleRSTFile.rst:415]: not sure about these double pairs of characters...

File: en_us/edx_style_guide/source/ExampleRSTFile.rst [hunk 46]
```diff
@@ -0,0 +1,471 @@
+.. _Anchor For ExampleRSTFile:
+
+#################
+Example .rst File
+#################
+
+If you work with edX documentation source files, you might find this file
+helpful as a reference. This file contains examples of .rst formatting and is
+not intended to be output.
+
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/ExampleRSTFile.rst:9]: I'm a little confused by this statement paired with the extra formatting around the markup, such as :: on line 18 so that the formatting for heading levels will show in HTML output. I don't think it's a big deal, but it's a bit confusing imo

**Comment by lamagnifica** [comment]
good to go! 
:+1:

File: en_us/edx_style_guide/source/DocTranslationProcess.rst (hunk 11 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: There are actually 2 "branch" actions, I guess: one to create the new language folder off master, and then another/others to create the "working branches" that I'm talking about here for actual translation work (for example, individual translators working on a branch to provide translations for particular files). Perhaps I'll pull out the initial process of creating the new language folder and say that WE own that? Then maybe once we create a new language folder on master they can start working? @mhoeber what do you think?

File: en_us/edx_style_guide/source/DocTranslationProcess.rst (hunk 44 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst:62]: "into" is correct here.

File: en_us/edx_style_guide/source/ExampleRSTFile.rst (hunk 46 above)
Comment (catong) [inline en_us/edx_style_guide/source/ExampleRSTFile.rst:9]: I'll look at this again to see if I can clarify. The sample file was not intended for output, then I wondered how translators would be able to get hold of this sample file if I didn't include it in the style guide.

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst (hunk 42 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst:102]: They needed extra pairs to show the double grave accent chars that they are described as having.

File: en_us/edx_style_guide/source/DocTranslationProcess.rst (hunk 43 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: It is linked. See output.

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst [hunk 47]
```diff
@@ -0,0 +1,104 @@
...
+   information and links to some resources, see :ref:`Work with edX
+   Documentation Source Files`.
+
+   In addition, you or another person on the translation team should
+   understand Github repositories and the Github pull request process. For
+   more information about using Github, see https://help.git.com.
+
+When you translate edX documentation source .rst files, follow these general
+guidelines. For examples of each type of element, see :ref:`the Example .rst
+file<Anchor For ExampleRSTFile>`.
```
Comment (mhoeber) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst:27]: I'd add please contact docs@edx.org before starting

File: en_us/edx_style_guide/source/DocTranslationProcess.rst (hunk 11 above)
Comment (mhoeber) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: working branch + "off of master."

Yes, I think we can own creating language folders

File: en_us/edx_style_guide/source/DocTranslationProcess.rst [hunk 48]
```diff
@@ -0,0 +1,65 @@
...
+   ``fr_ca``,``fr_fr``, ``zh_tw``, ``zh_cn``, and so on. The resulting folder
+   is your working source directory.
+
+#. Follow best practices for Github use. For information, see https://help.github.com.
+
+	* Create a working branch for your changes. Before making new branches,
+	  check that your local version of edx-documentation/master is up to date. 
+	* On your branch(es), make edits to the source files using your preferred
+	  text editor. 
+	* Save changes in the source files without renaming or moving the files.
```
Comment (mhoeber) [inline en_us/edx_style_guide/source/DocTranslationProcess.rst]: maybe not now -- but we should provide build instructions to test locally and address errors as much as possible.

**Comment by mhoeber** [comment]
@catong  this is great, couple more comments.  I'd go with this for now.  thanks

File: en_us/edx_style_guide/source/DocTranslationGuidelines.rst (hunk 47 above)
Comment (catong) [inline en_us/edx_style_guide/source/DocTranslationGuidelines.rst:27]: Done

**Comment by catong** [comment]
test this please

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/301
